tiledb/sm/query/readers/result_tile.h

```diff
@@ -213,12 +214,35 @@ class ResultTile {
     /*     CONSTRUCTORS & DESTRUCTORS    */
     /* ********************************* */
     TileData(
-        void* fixed_filtered_data,
-        void* var_filtered_data,
-        void* validity_filtered_data)
-        : fixed_filtered_data_(fixed_filtered_data)
-        , var_filtered_data_(var_filtered_data)
-        , validity_filtered_data_(validity_filtered_data) {
+        std::tuple<void*, ThreadPool::SharedTask> fixed_filtered_data,
+        std::tuple<void*, ThreadPool::SharedTask> var_filtered_data,
+        std::tuple<void*, ThreadPool::SharedTask> validity_filtered_data,
+        shared_ptr<FilteredData> filtered_data)
+        : fixed_filtered_data_(std::get<0>(fixed_filtered_data))
```

Comment:
Any reason you chose `std::tuple` over `std::pair`?  `.first` and `.second` is much less clunky than `std::get` in my opinion